assets/components/NavBar/NavBar.js

```diff
@@ -0,0 +1,57 @@
+
+import React, {Component} from 'react';
+import { Button } from './Button';
+import { MenuItems } from './MenuItems';
+import './NavBar.css';
+
+class NavBar extends Component {
+
+    state = {
+        clicked: false
+    }
+
+    handleClick = () => {
+        this.setState({clicked: !this.state.clicked})
+    }
+
+    render(){
+        return(
+            <nav className="NavBarItems">
+                <h1 className="navbar-logo">
+                    <a href="/" className="logo"><i className="fas fa-eye"></i></a>
+                </h1>
+                <div className="menu-icon" onClick={this.handleClick}>
+                    <i className={this.state.clicked ? "fas fa-times" : "fas fa-bars"}></i>
+                </div>
+                <ul className={this.state.clicked ? "nav-menu active" : "nav-menu"}>
+                    {MenuItems.map((item, index) => {
+                        return(
+                            <li key={index}>
+                                <a className={item.cName} href={item.url}>
+                                    {item.title}
+                                </a>
+                            </li>
+                        )
+                    })}
+                </ul>
+                    <Button>Login</Button>
+            </nav>
+        )
+    }    
+//    return (

```

Comment:
There's no need to keep things like this commented code in. Delete it, if it's to keep old code around "just in case", we have version control to see how things were in the past.